index.qmd

```diff
@@ -82,33 +74,41 @@ able to browse and share it, to tending to projects using the data.
 We plan to create four software products that run both separately and
 together:
 
-**sprout**: Grow your data in a structured and healthy way
-
-**propagate**: Submit requests for accessing specific data from a Data
-Resource\*
+-   [**sprout**](https://github.com/seedcase-project/seedcase-sprout):
+    Grow your data in a structured and healthy way
 
-**flower**: Catalogue and browse metadata on data in a Data Resource\*
+-   **propagate**: Submit requests for accessing specific data from a
+    Data Resource[^1]
 
-**garden**: Tend to multiple projects using data from a Data Resource\*
+-   **flower**: Catalogue and browse metadata on data in a Data Resource
 
-\*Data Resources are the central point in the Seedcase ecosystem. They
-are the place where data is stored and managed.
+-   **garden**: Tend to multiple projects using data from a Data
+    Resource
 :::
 
 ::: g-col-4
 ![](_extensions/seedcase-project/seedcase-theme/logos/software-product-logos.svg){fig-alt="The logos of the Seedcase software products."
 fig-align="right" width="200px"}
 :::
-:::
-:::
-:::
+:::::
+::::::
+:::::::
```

Comment:
This looks a bit crazy. @lwjohnst86 was this your formatter that did this?

Reply:
Same with L62 with **seven** colons?

Reply:
I know... I used the same VSCode formatter we've all been using, not sure why it did that and I didn't have time to look into it. It is weird for sure... I thought it was a mistake at first, so fixed it, than ran formatter again and it did it again 😠 💢

Reply:
Weird! Which formatter is this? I don't think it happens to me??